includes/Admin/Site_Health.php

```diff
@@ -0,0 +1,244 @@
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'wp_loaded';
+	}
+
+	/**
+	 * Adds the filters.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_filter( 'debug_information', [ $this, 'add_debug_information' ] );
+		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
+		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
+	}
+
+	/**
+	 * Adds debug information for Web stories.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param array $debugging_information The debugging information from Core.
+	 *
+	 * @return array The debugging information, with added information for Web stories.
+	 */
+	public function add_debug_information( array $debugging_information ) {
+		$extra_data = [
+			'web_stories'             => [
+				'label'       => esc_html__( 'Web Stories Constants', 'web-stories' ),
+				'description' => esc_html__( 'Debugging information for the Web Stories for WordPress.', 'web-stories' ),
+				'fields'      => [
+					'web_stories_version'        => [
+						'label'   => 'WEBSTORIES_VERSION',
+						'value'   => WEBSTORIES_VERSION,
+						'private' => false,
+					],
+					'web_stories_db_version'     => [
+						'label'   => 'WEBSTORIES_DB_VERSION',
+						'value'   => WEBSTORIES_DB_VERSION,
+						'private' => false,
+					],
+					'web_stories_amp_version'    => [
+						'label'   => 'WEBSTORIES_AMP_VERSION',
+						'value'   => WEBSTORIES_AMP_VERSION,
+						'private' => false,
+					],
+					'web_stories_cdn_url'        => [
+						'label'   => 'WEBSTORIES_CDN_URL',
+						'value'   => WEBSTORIES_CDN_URL,
+						'private' => false,
+					],
+					'web_stories_dev_mode'       => [
+						'label'   => 'WEBSTORIES_DEV_MODE',
+						'private' => false,
+						'value'   => $this->get_formatted_output( WEBSTORIES_DEV_MODE ),
+						'debug'   => WEBSTORIES_DEV_MODE,
+					],
+					'web_stories_theme_support'  => [
+						'label'   => 'Theme supports',
+						'value'   => $this->get_formatted_output( current_theme_supports( 'web-stories' ) ),
+						'private' => false,
+					],
+					'web_stories_libxml_version' => [
+						'label'   => 'libxml Version',
+						'value'   => LIBXML_DOTTED_VERSION,
+						'private' => false,
+					],
+				],
+			],
+			'web_stories_experiments' => [
+				'label'       => esc_html__( 'Web Stories Experiments', 'web-stories' ),
+				'description' => esc_html__( 'Information on enabled and disabled experiments for the Web Stories plugin.', 'web-stories' ),
+				'fields'      => [],
+			],
+		];
+
+		foreach ( $this->experiments->get_experiments() as $experiment ) {
+			$extra_data['web_stories_experiments']['fields'][ $experiment['name'] ] = [
+				'label'   => $experiment['label'],
+				'value'   => $this->get_formatted_output( $this->experiments->is_experiment_enabled( $experiment['name'] ) ),
+				'private' => false,
+			];
+		}
```

Comment:
Using `$this->experiments->get_enabled_experiments()` makes the list a bid tidier.

Not interested in disabled ones as they don't have any effect.

```suggestion
		foreach ( $this->experiments->get_enabled_experiments() as $experiment ) {
			$extra_data['web_stories_experiments']['fields'][ $experiment['name'] ] = [
				'label'   => $experiment['label'],
				'value'   => $this->get_formatted_output( true ),
				'private' => false,
			];
		}
```

Reply:
We just want enabled experiments, is that correct?

Reply:
Yes 👍

Reply:
Tweaked in https://github.com/google/web-stories-wp/pull/7570/commits/0b0e2cad6be248e8df3e18c5faaebc723f4ae53d